Sources/PayPalNativePayments/PayPalNativeCheckoutRequest.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+/// Used to configure options for approving a PayPal native order
+public struct PayPalNativeCheckoutRequest {
+
```

Comment:
NXO exposes "billing agreement token" and "vault approval session ID" we could technically also send. I opted not to, until we spend time fully testing those features.

Raising here in case folks feel differently.

Reply:
Agree with waiting to fully test those to add them. Would also be nice to know how much/if they are requested by merchants. I wonder if the MXO team can pull this data to see if it will be required as merchants transition from the direct MXO integration.

Reply:
That's a great callout for us to keep in mind as we need to migrate MXO direct merchants over to PPCP.

Reply:
+1 Keeping the scope tighter makes sense to start.